playbooks/02-infra.yml

```diff
@@ -86,6 +86,22 @@
       ansible.builtin.include_role:
         name: cert_manager
 
+    # Deploy Sushy Emulator into OCP
+    - name: Deploy Sushy Emulator service pod
+      when: cifmw_use_sushy_emulator | default(false) | bool
+      vars:
+        cifmw_sushy_emulator_hypervisor_address: "{{ hostvars['controller'].ansible_host }}"
+        cifmw_sushy_emulator_hypervisor_target: controller
+        cifmw_sushy_emulator_install_type: ocp
+      ansible.builtin.include_role:
+        name: sushy_emulator
+
+    - name: Setup Libvirt on controller
+      when: cifmw_use_libvirt_extracted | default(false) | bool
```

Comment:
Not sure what var to use here, cifmw_use_libvirt is already used in the reproducer scenario so using cifmw_use_libvirt_extracted as a placeholder until discussion.